frontend/src/components/Table/Table.tsx

```diff
@@ -370,7 +370,7 @@ const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
   };
 
   const renderHeaderCell = (column: ColumnInstanceWithProps<T>) => {
-    return (
+    return filterable ? (
```

Comment:
if the columns aren't filterable, don't try and wrap the header with the sortable/filterable components that require a formik wrapper.